betterlint.gemspec

```diff
@@ -24,10 +24,12 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 3.0"
 
-  s.add_dependency "rubocop", "~> 1.62.0"
-  s.add_dependency "rubocop-graphql", "~> 1.5.0"
-  s.add_dependency "rubocop-performance", "~> 1.21.0"
-  s.add_dependency "rubocop-rails", "~> 2.24.0"
-  s.add_dependency "rubocop-rake", "~> 0.6.0"
-  s.add_dependency "rubocop-rspec", "~> 2.28.0"
+  s.add_dependency "rubocop", "~> 1.62"
+  s.add_dependency "rubocop-graphql", "~> 1.5"
+  s.add_dependency "rubocop-performance", "~> 1.21"
+  s.add_dependency "rubocop-rails", "~> 2.24"
+  s.add_dependency "rubocop-rake", "~> 0.6"
+  s.add_dependency "rubocop-rspec", ">= 2.28"
+  s.add_dependency "rubocop-capybara", "~> 2.21"
+  s.add_dependency "rubocop-factory_bot", "~> 2.26"
```

Comment:
There are rules in the default config which make it impossible to run without them.  It seems they are currently, accidental undeclared dependencies.  I only added them because I was forced to.

Errors were:
```
Error: The `RSpec/Capybara/FeatureMethods` cop has been removed since this cop has migrated to `RSpec/Dialect`. Please use `RSpec/Dialect` instead.
(obsolete configuration found in config/default.yml, please update it)
`RSpec/Capybara/*` has been extracted to the `rubocop-capybara` gem.
(obsolete configuration found in config/default.yml, please update it)
```
and
```
Error: unrecognized cop or department FactoryBot/AssociationStyle found in config/default.yml
unrecognized cop or department FactoryBot/ConsistentParenthesesStyle found in config/default.yml
unrecognized cop or department FactoryBot/SyntaxMethods found in config/default.yml
```

Reply:
Actually, I did remove the Capybara rule it complained about, because the replacement suggestion didn't make much sense to me, saying both `RSpec/Dialect`, and `rubocop-capybara`, which don't sound related.

But since it had been there it was clear that `rubocop-capybara` was intended to be a dependency.  I don't mind removing it, as I don't think there are any rules remaining that relate to it.

The factory-bot rules are still in the config though, so I think the gem is still required.

Reply:
The GitHub Actions workflows are [now passing](https://github.com/VitalConnectInc/betterlint/actions)!  This is ready for review.